src/datasource/components/Logs/LogsCell.tsx

```diff
@@ -0,0 +1,325 @@
+                ...style
+            }}>
+            <div
+                style={{
+                    paddingLeft: '10px',
+                    paddingRight: '15px'
+                }}
+            >
+                <Button
+                    size={'sm'}
+                    variant={'secondary'}
+                    icon={getFoldIcon()}
+                    onClick={() => onClick(rowIndex)}
+                />
+            </div>
+            <div>
+                {formattedTime}
+            </div>
+        </div>
+    );
+}
+
+const FieldCell = () => {
+    return (<></>);
+}
+
+const HeaderCell = (column: LogColumn, style) => {
+    // TODO: Handle field types
+    // const log = data.logs[rowIndex];
+    // const _timeField = data.timeField;
+
+    // TODO: Implement sorting?
+    return (
+        <div
+            style={{
+                paddingLeft: column.logColumnType === LogColumnType.TIME ? '10px' : '0px',
+                ...style
+            }}
+        >
+            {column.logColumnType === LogColumnType.TIME ? 'Time' : 'Document'}
+        </div>
+    )
+}
+
+// Either expands or collapses the row, depending on the state its currently in
+const invertRow = (expandedRows: boolean[], rowIndex: number): boolean[] => {
+    expandedRows[rowIndex] = !expandedRows[rowIndex]
+    return expandedRows
+}
+
+// If the row isn't being expanded, then shrink it back to its original size.
+// NOTE: Because of how we handle it down below, "shrink it to its original size"
+// effectively means set it to 0.
+const shrinkRows = (expandedRows: boolean[], rowIndex: number, setSize: (index: number, value: number) => void): boolean => {
+    if (!expandedRows[rowIndex]) {
+        setSize(rowIndex, 0);
+        return true;
+    }
+    return false;
+}
+
+
+
+const LogCell = ({ columnIndex, rowIndex, style, data }) => {
+    const log = data.logs[rowIndex];
+    const timestamp = data.timestamps[rowIndex];
+    const column = data.columns[columnIndex];
+    const setExpandedRowsAndReRender = data.setExpandedRowsAndReRender
+    const expandedRows = data.expandedRows
+    const datasourceUid = data.datasourceUid
+    const datasourceName = data.datasourceName
+    const { setSize } = getLogTableContext();
+    const darkModeEnabled = useTheme2().isDark ;
+
+
+    // TODO: Ignoring for now as these will be used in a future pass
+    // const _timeField = data.timeField;
+    // const _setColumns = data.setColumns
+
+    const handleOnClick = (rowIndex: number): any => {
+        const newExpandedRows = invertRow(expandedRows, rowIndex);
+        shrinkRows(newExpandedRows, rowIndex, setSize);
+        setExpandedRowsAndReRender([...newExpandedRows], rowIndex);
+    }
+
+    // Handle drawing the borders for the entire row
+    // Only draw a borderon the left if we're on the left-most cell
+    if (columnIndex === 0) {
+        style['borderLeft'] = darkModeEnabled ? '1px solid rgb(71, 71, 71)' : '1px solid rgba(36, 41, 46, 0.3)';
```

Comment:
We should move to a stylesheet based approach for these if possible - it should result in faster rendering performance than doing inline styling, and is generally a bit easier to maintain long-term.

Reply:
This border color is inconsistent with Grafana's.

![image](https://github.com/slackhq/slack-kaldb-app/assets/771133/1f0984ef-530a-45b5-9bd5-9ac421755fbf)